CBLClient/Apps/CBLTestServer-Dotnet/TestServer/Router.cs

```diff
@@ -313,7 +313,7 @@ internal static void Handle([NotNull]Uri endpoint, [NotNull]Stream body, [NotNul
         {
             MemoryMap.Clear();
             response.WriteEmptyBody(HttpStatusCode.OK);
-            // return;
+            return;
```

Comment:
unnecessary return statement

Reply:
no , this is throwing error on flushMemory() without return. We should have return . I see Jenkins is having same error now . This should fix jenkins failures